markdown/renderer.go

```diff
@@ -55,6 +56,12 @@ func WithUnderlineHeadings() Option {
 	}
 }
 
+func WithHardWraps() Option {
```

Comment:
Can we add comment?

Also ... technically we talk about soft wraps, no?
```suggestion
func WithSoftWraps() Option {
```

Reply:
Sure! Will add a comment and rename it! So everything hardWrap -> softWrap.